src/core/zowe/core_for_zowe_sdk/validators.py

```diff
@@ -18,29 +18,25 @@ def validate_config_json(path_config_json: str, path_schema_json: str):
     """
     Function validating that zowe.config.json file matches zowe.schema.json.
 
-
     Parameters
     ----------
-        path_config - requires absolute path to zowe.config.json
+        path_config: str
+            Absolute path to zowe.config.json
 
-        path_schema - requires absolute path to zowe.schema.json
+        path_schema: str
+            Absolute path to zowe.schema.json
 
     Returns
     -------
-        Confirms if the config.json matches schema or provides appropriate details if it doesn't.
+        Provides details if config.json doesn't match schema.json, otherwise it returns None.
     """
 
-    config_json = ""
-    schema_json = ""
-
-    with open(path_config_json) as f:
-        config_json = commentjson.load(f)
+    with open(path_config_json) as file:
+        config_json = commentjson.load(file)
 
-    with open(path_schema_json) as f:
-        schema_json = commentjson.load(f)
+    with open(path_schema_json) as file:
+        schema_json = commentjson.load(file)
 
     result = validate(instance=config_json, schema=schema_json)
```

Comment:
Is `config_json` or `schema_jsopn`even  defined here?
I would propose this to be rewritten like this:
```
with open(path_config_json) as config_file, open(path_schema_json) as schema_file:
    config_json = commentjson.load(config_file)
    schema_json = commentjson.load(schema_file)
    return validate(instance=config_json, schema=schema_json)
```

Reply:
You don't need to check `if result` if you anyway would return None `if not result`

Reply:
A 'with' statement does not create a scope, for the 'if result' I'll just return what you suggested.